src/filters/filters.jl

```diff
@@ -46,7 +46,7 @@ See [test/filter.jl](https://github.com/JuliaIO/HDF5.jl/blob/master/test/filter.
 module Filters
 
 # builtin filters
-export Deflate, Shuffle, Fletcher32, Szip, NBit, ScaleOffset
+export Deflate, Shuffle, Fletcher32, Szip, NBit, ScaleOffset, ExternalFilter
```

Comment:
If we export `ExternalFilter` we probably should redefine show so that it doesn't print out 
`HDF5.Filters.UnkownFilter`

Reply:
Maybe we should just switch the alias then. `UnknownFilter` is an alias for `ExternalFilter` ?

Reply:
Agreed, if this is exported that would be easier and make more sense.

Reply:
Done in [3f5ec32](https://github.com/JuliaIO/HDF5.jl/pull/911/commits/3f5ec321f73322ecd566b1d206a028d82b9ad286)